src/machine/machinectl.c

```diff
@@ -63,7 +63,7 @@
 
 static char **arg_property = NULL;
 static bool arg_all = false;
-static bool arg_value = false;
+static BusPrintPropertyFlags arg_print_flags = 0;
 static bool arg_full = false;
```

Comment:
any reason to keep `arg_all` around in this one?

Reply:
`arg_all` is also used for listing machines and images, and in those cases, `bus_print_property()` is not used. I think it is better to keep `arg_all` for readability. But if you do not like it, all `arg_all` can be replaced with `FLAGS_SET(arg_print_flags, BUS_PRINT_PROPERTY_SHOW_EMPTY)`.

Reply:
it's ok as is, makes sense to me, thanks